packages/amplify-category-storage/src/__tests__/provider-utils/awscloudformation/cdk-stack-builder/ddb-stack-transform.test.ts

```diff
@@ -26,8 +26,8 @@ jest.mock('fs-extra', () => ({
 }));
 
 jest.mock('path', () => ({
-  join: jest.fn().mockReturnValue('mockjoinedpath'),
-  resolve: jest.fn().mockReturnValue('mockjoinedpath'),
+  join: jest.fn().mockReturnValue('src/__tests__/mockjoinedpath'),
+  resolve: jest.fn().mockReturnValue('src/__tests__/mockjoinedpath'),
```

Comment:
so that we don't trigger rebuilds.

Reply:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/5849952/217149716-e4f162f8-0191-4062-a4e0-6c9c0276d2ed.png">